libraries/AP_HAL_ChibiOS/hwdef/MatekH743/hwdef.dat

```diff
@@ -175,26 +178,31 @@ define HAL_STORAGE_SIZE 16384
 define STORAGE_FLASH_PAGE 14
 
 # spi devices
-SPIDEV ms5611    SPI3 DEVID1 MS5611_CS    MODE3 20*MHZ 20*MHZ
-SPIDEV mpu6000  SPI1 DEVID1 IMU1_CS MODE3  1*MHZ  4*MHZ
+SPIDEV mpu6000   SPI1 DEVID1 IMU1_CS      MODE3  1*MHZ  4*MHZ
 SPIDEV icm20602  SPI4 DEVID1 IMU2_CS      MODE3  1*MHZ  4*MHZ
-SPIDEV osd      SPI2 DEVID4 MAX7456_CS  MODE0 10*MHZ 10*MHZ
+SPIDEV osd       SPI2 DEVID4 MAX7456_CS   MODE0 10*MHZ 10*MHZ
+
+# SPI3 external connections
+SPIDEV pixartflow   SPI3 DEVID1  EXT_CS1  MODE3   2*MHZ  2*MHZ
 
 # no built-in compass, but probe the i2c bus for all possible
 # external compass types
 define ALLOW_ARM_NO_COMPASS
 define HAL_COMPASS_DEFAULT HAL_COMPASS_NONE
 define HAL_PROBE_EXTERNAL_I2C_COMPASSES
-define HAL_I2C_INTERNAL_MASK 1
+define HAL_I2C_INTERNAL_MASK 0
 define HAL_COMPASS_AUTO_ROT_DEFAULT 2
 
 # two IMUs. We put icm20602 first as we can sample accel at 4kHz
 IMU Invensense SPI:icm20602 ROTATION_ROLL_180_YAW_270
 IMU Invensense SPI:mpu6000 ROTATION_ROLL_180_YAW_270
 define HAL_DEFAULT_INS_FAST_SAMPLE 1
 
-# one BARO
-BARO MS56XX SPI:ms5611
+# MS5611 integrated on I2C2 bus, multiple possible choices for external barometer
+BARO MS56XX I2C:0:0x77
+BARO BMP280 I2C:0:0x76
+BARO BMP388 I2C:0:0x76
+two BARO
```

Comment:
stray comment? (if wanted, needs # at start of line)
also, do you want to leave the SPI baro in as well so that the original prototype boards still work?

Reply:
please also squash to one commit, and prefix commit msg with "HAL_ChibiOS: "